src/javascripts/embed.coffee

```diff
@@ -52,7 +53,10 @@ class Iframe
 
     hsh(string.charCodeAt(i)) for i in [0..string.length]
 
-    return hash.toString(16).substring(1)
+    potentialId = hash.toString(16).substring(1)
+    return potentialId if _.isNumber(potentialId.substring(0, 1))
```

Comment:
Can't we use `replace` here?

`"34dfhajkldsfhjsdkflh".replace(/^[0-9]/, 'a')` or on your code:

`hash.toString(16).substring(1).replace(/^[0-9]/, 'a')`

But to be honest, not sure what `toString(16)` followed by `substring(1)` is doing here. Maybe I do expect the wrong input.

Reply:
Thanks for your input @marcelgo 🙂  Maybe @benzimmer has some insights/feelings about this?